2020-10-25 16:49:08.766 - [Info] MITM Version: 1.26
2020-10-25 16:49:08.773 - [Info] Auto Access Enabled: true
2020-10-25 16:49:08.773 - [Debug] [Init] Auto Access Barrier: true
2020-10-25 16:49:08.795 - [Info] SSH man-in-the-middle server for 172.20.0.202 listening on 0.0.0.0:10001
2020-10-25 16:51:33.681 - [Debug] [Connection] Attacker connected: 193.105.134.45 | Client Identification: SSH-2.0-OpenSSH_6.0
2020-10-25 16:51:34.844 - [Debug] [Auth] Attacker 193.105.134.45 trying to authenticate with "none"
2020-10-25 16:51:34.845 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-25 16:51:35.124 - [Debug] [Auth] Attacker 193.105.134.45 trying to authenticate with "password"
2020-10-25 16:51:35.125 - [Debug] [Auto Access] Attacker: 193.105.134.45, Threshold: 1, Attempts: 1
2020-10-25 16:51:35.125 - [Debug] [Auto Access] Compromising the honeypot
2020-10-25 16:51:35.125 - [Debug] [Auto Access] Adding the following credentials: "123321:111111"
2020-10-25 16:51:35.741 - [Debug] [Auth] Valid credentials - Password Authentication
2020-10-25 16:51:35.741 - [Debug] [LXC] Attempting to connect to the honeypot: 172.20.0.202
2020-10-25 16:51:35.833 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2020-10-25 16:51:36.947 - [Debug] [Connection] Attacker closed connection
2020-10-25 16:51:37.112 - [Debug] [LXC] Container's OpenSSH server ended connection
2020-10-25 16:51:37.113 - [Debug] [LXC] Container's OpenSSH server closed connection
2020-10-25 16:52:20.676 - [Debug] [Connection] Attacker connected: 185.2.81.183 | Client Identification: SSH-2.0-sshlib-0.1
2020-10-25 16:52:22.215 - [Debug] [Auth] Attacker 185.2.81.183 trying to authenticate with "password"
2020-10-25 16:52:22.216 - [Debug] [Auth] Attacker authentication error: Invalid credentials - Password Authentication Failure
2020-10-25 16:52:22.217 - [Debug] [Auth] Attacker: 185.2.81.183 has so far made 1 attempts. Remaining: 5 attempts
2020-10-25 16:52:22.435 - [Debug] [Connection] Attacker closed the connection
2020-10-25 16:53:12.300 - [Debug] [Connection] Attacker connected: 195.3.147.47 | Client Identification: SSH-2.0-PuTTY_Release_0.58
2020-10-25 16:53:13.871 - [Debug] [Auth] Attacker 195.3.147.47 trying to authenticate with "none"
2020-10-25 16:53:13.872 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2020-10-25 16:53:14.132 - [Debug] [Auth] Attacker 195.3.147.47 trying to authenticate with "password"
2020-10-25 16:53:14.133 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2020-10-25 16:53:14.134 - [Debug] [Auth] Attacker: 195.3.147.47 has so far made 1 attempts. Remaining: 5 attempts
2020-10-25 16:53:14.392 - [Debug] [Connection] Attacker closed the connection
